.github/workflows/migration-check.yaml

```diff
@@ -53,14 +53,31 @@ jobs:
           features: postgres
           args: "--no-default-features"
 
-      - name: Verify `diesel migration run`
+      - uses: baptiste0928/cargo-install@v2.2.0
+        with:
+          crate: just
+          args: "--no-default-features"
+
+      - name: Verify `diesel migration run for v1`
+        shell: bash
+        env:
+          DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres
```

Comment:
You can move this to just after the `runs-on` line to avoid repeating it multiple times:

```yaml
jobs:
  migration_verify:
    name: Verify consistency of migrations with `schema.rs` file
    runs-on: ubuntu-latest

    env:
      DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres
```